imports/api/processes/process.methods.js

```diff
@@ -1,58 +1,21 @@
-};
-
-const getProcessLeaveText = (firstName, processTitle, processId) => {
-  return `Hi ${firstName},\n\nThis is a confirmation email to inform you that you have successfully left the study process called "${processTitle}".\nIf you want to join the process again, you can do so here at the process page: ${siteUrl}process/${processId}.\n\nKind regards,\n${publicSettings.name} Team`;
-};
-
-const getMeetingAttendText = (
-  firstName,
-  occurence,
-  processTitle,
-  processId
-) => {
-  return `Hi ${firstName},\n\nThis is a confirmation email to inform you that you have successfully registered your attendance for the meeting on ${occurence.startDate} at ${occurence.startTime} as part of the study process called "${processTitle}".\nMay there be any changes to your attendance, please update and inform your friends at the process page: ${siteUrl}process/${processId}.\n\nYou are encouraged to follow the updates, register to attend meetings and join the discussion at this page.\n\nWe look forward to your participation.\n${publicSettings.name} Team`;
-};
-
-const getMeetingUnattendText = (
-  firstName,
-  occurence,
-  processTitle,
-  processId
-) => {
-  return `Hi ${firstName},\n\nThis is a confirmation email to inform you that we have successfully removed your attendance from the meeting on ${occurence.startDate} at ${occurence.startTime} as part of the study process called "${processTitle}".\nMay there be any changes to your attendance, please update and inform your friends at the process page: ${siteUrl}process/${processId}.\n\nYou are encouraged to follow the updates, register to attend meetings and join the discussion at this page.\n\nWe look forward to your participation.\n${publicSettings.name} Team`;
-};
-
-const getInviteToPrivateProcessText = (
-  firstName,
-  processTitle,
-  processId,
-  processAdmin
-) => {
-  return `Hi ${firstName},\n\nThis is an email to invite you to a private process entitled ${processTitle} created by ${processAdmin}.\n\nIf you wish to accept this invite and join the process, simply go to the process page and click the "Join" button: ${siteUrl}process/${processId}.\n\nPlease bear in mind that you have to have an account at the ${publicSettings.name} App, or create one, with this email address to which you received this email.\n\nYou are encouraged to follow the updates, register to attend meetings and join the discussion at this page.\n\nWe look forward to your participation.\n${publicSettings.name} Team`;
-};
-
-const compareForSort = (a, b) => {
-  const dateA = new Date(a.endDate);
-  const dateB = new Date(b.endDate);
-  return dateA - dateB;
-};
-
 Meteor.methods({
-  createProcess(formValues, imageUrl, isPrivate = false) {
+  async createProcess(formValues, imageUrl, isPrivate = false) {
```

Comment:
reason this is async and others not?

Reply:
you got this thanx, I was getting an error from that method and while I was trying things I made it async and then I just forget it async, we can take it back to normal ...

Reply:
Yeah that error I'm getting too, and a strange one that we can address later. It's some other outdated code...